kong/plugins/prometheus/exporter.lua

```diff
@@ -313,6 +333,66 @@ local function log(message, serialized)
       metrics.kong_latency:observe(kong_proxy_latency, labels_table_latency)
     end
   end
+
+  if serialized.ai_metrics then
+    for _, ai_plugin in pairs(serialized.ai_metrics) do
+      local cache_status
+      if ai_plugin.cache and ai_plugin.cache.cache_status then
+        cache_status = ai_plugin.cache.cache_status
+      end
+
+      local vector_db, embeddings_provider, embeddings_model
+      if ai_plugin.cache then
+        if ai_plugin.cache.vector_db then
+          vector_db = ai_plugin.cache.vector_db
+        end
+
+        if ai_plugin.cache.embeddings_provider then
+          embeddings_provider = ai_plugin.cache.embeddings_provider
+        end
+
+        if ai_plugin.cache.embeddings_model then
+          embeddings_model = ai_plugin.cache.embeddings_model
+        end
```

Comment:
```suggestion
        vector_db = ai_plugin.cache.vector_db

        embeddings_provider = ai_plugin.cache.embeddings_provider

        embeddings_model = ai_plugin.cache.embeddings_model
```

Reply:
I'm also bit worried about the fact that we are accessing members directly like `ai_plugin.cache.vector_db` so that a typo or misalign of names could silently cause it to failure and is hard to be caught by eye, and it's not covered by tests. not blocker for this PR, but we need to improve on this in the future.